documentation/modules/ROOT/pages/configuration/outbox-event-router.adoc

```diff
@@ -90,69 +89,69 @@ This result was achieved with the link:#configuration-options[default configurat
 |Group
 |Description
 
-|`table.field.event.id`
+|[[configuration-property-table-field-event-id]]<<configuration-property-table-field-event-id, `table.field.event.id`>>
 |`id`
 |Table
 |The column which contains the event ID within the outbox table
 
-|`table.field.event.key`
+|[[configuration-property-table-field-event-key]]<<configuration-property-table-field-event-key, `table.field.event.key`>>
 |`aggregateid`
 |Table
 |The column which contains the event key within the outbox table; when this is set the value of this column will be used as a Kafka message key
 
-|`table.field.event.type`
+|[[configuration-property-table-field-event-type]]<<configuration-property-table-field-event-type, `table.field.event.type`>>
```

Comment:
I think this can be removed, right? It won't serve any purpose any longer, now that event type is solely an "additional field" as any other the SMT has no awareness of.